### ISSUE TITLE
Container cannot start.

### ISSUE THREAD

**Issue by Pandorin** [awesto/cookiecutter-django-shop#35]
```
$ cookiecutter https://github.com/awesto/cookiecutter-django-shop
$ docker-compose up --build -d
```
```
webapp_1 | File "/web/dveri4you/models.py", line 6, in <module>
webapp_1 | from shop.models.order import BaseOrderItem
webapp_1 | File "/usr/local/lib/python3.6/site-packages/shop/models/order.py", line 14, in <module>
webapp_1 | from ipware.ip import get_ip
webapp_1 | ImportError: cannot import name 'get_ip'
```
Few hours ago released [django-ipware 3.0.0](https://github.com/un33k/django-ipware/releases/tag/3.0.0) and changed [get_ip()](https://github.com/un33k/django-ipware/blob/3.0.0/ipware/ip.py) method (_Remove deprecated logic_).
`django-ipware = "*"` in [Pipfile](https://github.com/awesto/cookiecutter-django-shop/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/Pipfile#L14).

**Comment by Pandorin** [comment]
[Fix bug with ipware.ip import get_ip #825](https://github.com/awesto/django-shop/pull/825).

**Comment by jeffykle** [comment]
In the meantime 
`pip install django-ipware==2.1.0`

**Comment by jrief** [comment]
This has been fixed with django-shop 1.2.3
Thanks for informing me.
